portfolio/src/main/webapp/index.html

```diff
@@ -34,6 +34,20 @@ <h2>Projects</h2>
         the maze. The maze was created using a 2D array of images. The sprite could move through the maze
         by looking at the next image and seeing if it was "empty" or a "wall". The game also required the
         user to retreive a key before being able to complete the level. </p>
+
+    <h2>Academic Achievements</h2>
+    <ul>
+        <li> Finished the 2019-2020 school year at NC A&T SU with an average GPA  of 4.0.</li>
+        <li> Chancellor's List for Fall 2019 (Chancellor's and Dean's List was not applied to Spring 2020)</li>
+        <li> Accepted into the Honors Program in January 2020</li>
+        <li> Techical Courses taken: COMP 163(Intro to Programming), COMP 167(Computer Program Design), and COMP 180(Discrete Structures)</li>
+        <li> Technical Courses taking next semester: COMP 280(Data Structures)</li>
+        <li> ACM-W President for 2020-2021 school year</li>
+        <li> etc....</li>
+    </ul>
+
+    <!-- Fake link, goes nowhere -->
+    <p> View my transcript <a href="#">here</a></p>
```

Comment:
Are you leaving this for a separate change or intentionally not adding it?

Reply:
Intentionally. I don't actually know how to link my transcript but it seems like it would go here. Do you believe I should remove it?

Reply:
Nope, fake links are fine, just explain why in the comment.

As for linking it, I would upload the pdf like you did with the images and then just make the link to "/transcript.pdf".  That *should* work?